wallet/src/de/schildbach/wallet/ui/dashpay/work/UpdateProfileWorker.kt

```diff
@@ -98,6 +101,8 @@ class UpdateProfileWorker(context: Context, parameters: WorkerParameters)
                     KEY_USER_ID to profileRequestResult.userId
             ))
         } catch (ex: Exception) {
+            FirebaseCrashlytics.getInstance().log("Failed to create/update profile: retrieve password")
```

Comment:
The error here would be "broadcast state transition" instead of "retrieve password"

Reply:
Fixed on https://github.com/dashevo/dash-wallet/pull/584/commits/ed92e5037076b1bb16e6530429cc57432bc3f341